src/styles/components/_calendar.scss

```diff
@@ -0,0 +1,50 @@
+@import "../base/base";
+@import "../abstracts/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $width;
+  gap: $gap;
+  padding: $padding;
+
+  @for $i from 1 through 31 {
```

Comment:
You don't need this loop. When you change class for all cells to be the same you can style them with only one selector